File: BasicsWithJava/MultipleActivity/app/src/main/java/com/sumere/multipleactivity/MainActivity.java
package com.sumere.multipleactivity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    String name = "";
    EditText nameText;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        nameText = findViewById(R.id.nameTextA1);
    }
    public void changeActivity(View view){
        name = nameText.getText().toString();
        Intent intent = new Intent(this,MainActivity2.class);
        intent.putExtra("Input",name);
        startActivity(intent);
    }
}

File: CoroutineExceptionKotlin/app/src/main/java/com/sumere/coroutineexceptionkotlin/MainActivity.kt
package com.sumere.coroutineexceptionkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.lifecycle.lifecycleScope
import kotlinx.coroutines.launch
import kotlin.Exception

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        lifecycleScope.launch {
            try {

                    throw Exception("Error")

            }catch (e: Exception){
                e.printStackTrace()
            }
        }
    }
}